docs/ref/forms/fields.txt

```diff
@@ -380,6 +380,12 @@ For each field, we describe the default widget used if you don't specify
        If ``True`` (default), the value will be stripped of leading and
        trailing whitespace.
 
+    .. attribute:: empty_value
+
+       .. versionadded:: 1.11
+
+       The value to use to represent "empty". Defaults to the empty string.
```

Comment:
> Maybe you could add another sentence giving an example use case for customizing it.

TBH, I don't really have one.

The API was added to the `CharField` as it appeared to me to be an elegant way to handle the model form/admin problem. I have no direct, personal use for the API except the existing use by the model forms. I only documented it because it seemed like a nice to have for the PR.

The API was taken directly from `TypedChoiceField`. That field has the following additional words:

> Defaults to the empty string; `None` is another common choice here.

Is this extra blurb sufficient?

I'm wiling to write something here but it would just be a fabrication not based on a real life use case.